@xen-orchestra/xva-generator/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@xen-orchestra/xva-generator",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+  },
+  "author": "",
+  "license": "ISC",
+  "private": true,
```

Comment:
You will likely want to change this to `true`.

And after you will need to re-run `normalize-packages`.